src/app/plugins/ta.plugin.ts

```diff
@@ -0,0 +1,158 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType, Maybe } from '../../common/types';
+import { GuildChannel, GuildMember, TextChannel, Role } from 'discord.js';
+
+export class TaPlugin extends Plugin {
+  public name: string = 'TA Plugin';
+  public description: string = 'Allows TAs to register for classes.';
+  public usage: string = 'ta register <class_name> | ta ask <question>';
```

Comment:
class_name shouldn't be necessary; we can have the TA register within the class channel and then infer it from message.channel

Reply:
yea i see that's what you do... so just change this help message by removing <class_name>

Reply:
Sorry, old usage. I changed the way it registered TAs